src/client/components/definition/definitionData.js

```diff
@@ -0,0 +1,36 @@
+import {
+  DEFINITION_OPEN,
+  DEFINITION_HIGH,
+  DEFINITION_LOW,
+  DEFINITION_MARKET_CAP,
+  DEFINITION_VOLUME,
+  DEFINITION_DIVIDEND,
+  DEFINITION_PE_RATIO,
+  DEFINITION_ROE,
+  DEFINITION_FCFY
+} from '../../../constants';
+
+module.exports = Object.freeze({
```

Comment:
It probably makes sense to do this in an (alphabetized) array, so that it becomes more straightforward to iterate over the terms in the Glossary component.
```
[
  {
    title: 'Dividend (Yield %)',
    definition: 'An amount of the company\'s profits...'
  },
  {
    title: 'Free Cash Flow Yield (abbrev. as FCFY)',
    definition: 'A comparison between free cash flow per share and share price...'
  },
  ...
]
```
I don't think the DEFINITION_ constants will be necessary after going to this approach (but if you do for some reason still need them, I'd change DEFINITION_ to DEFINITION_TERM_, since the DEFINITION_s are terms and not the actual definitions of the terms)